src/MediaRental/DatabaseSupport.java

```diff
@@ -32,126 +32,156 @@ public DatabaseSupport()
 			System.err.println("Unable to load driver.");
 			E.printStackTrace();
 		}
+		return DriverManager.getConnection(DB_URL, USER, PASSWORD);
+	}
+
+	public boolean putCustomer(Customer c)
```

Comment:
I dont think that we should put the customer in the database with all of the transactions on it. A customer should be created, then transactions should be added to it.